app/app.go

```diff
@@ -915,14 +915,17 @@ func NewExocoreApp(
 	app.mm.SetOrderBeginBlockers(
 		upgradetypes.ModuleName,    // to upgrade the chain
 		capabilitytypes.ModuleName, // before any module with capabilities like IBC
-		epochstypes.ModuleName,     // to update the epoch
-		feemarkettypes.ModuleName,  // set EIP-1559 gas prices
-		evmtypes.ModuleName,        // stores chain id in memory
-		slashingtypes.ModuleName,   // TODO after reward
-		evidencetypes.ModuleName,   // TODO after reward
-		stakingtypes.ModuleName,    // track historical info
-		ibcexported.ModuleName,     // handles upgrades of chain and hence client
-		authz.ModuleName,           // clear expired approvals
+		// being block of oracle will fill params cache, need to be put before epochs will use the params
```

Comment:
```suggestion
		// begin block of oracle will fill params cache, need to be put before epochs will use the params
```

Why is epochs using the params?

Reply:
hooks.AfterEpochEnd -> operator.UpdateVotingPower -> oracle.GetMultipleAssetsPrices -> k.GetParamsFromCache